examples/all-clusters-app/esp32/main/DeviceWithDisplay.cpp

```diff
@@ -520,6 +550,24 @@ void SetupPretendDevices()
     app::Clusters::ColorControl::Attributes::CurrentHue::Set(1, 200);
     AddAttribute("Current Saturation\n", "150");
     app::Clusters::ColorControl::Attributes::CurrentSaturation::Set(1, 150);
+
+    AddDevice("Window Covering");
+    AddEndpoint("1");
+    AddCluster("Window Covering");
+    AddAttribute("Current Lift", "5");
```

Comment:
How this "Current Lift" via text resolution works ?

Reply:
This is treated as a string tag for the UI.